Sources/MongoSwift/WriteConcern.swift

```diff
@@ -158,14 +174,30 @@ public class WriteConcern: Codable {
         }
         try container.encodeIfPresent(self.journal, forKey: .j)
     }
-
-    /// Cleans up internal state.
-    deinit {
-        guard let writeConcern = self._writeConcern else {
-            return
+    /**
+    *  Creates a new `mongoc_write_concern_t` based on this `WriteConcern` and passes it to the provided closure.
```

Comment:
nit: this asterisk and the next one should both be forward one space, aligned with the first asterisk in the opening line.